yetanotherco/aligned_layer
# feat: broken sockets metric in batcher

**Changes**
Adds metrics for the broken sockets when sending responses in the batcher.

**Testing**
Set up a local testnet as usual, and:
- Run metrics: `make run_metrics`
- Start sending proofs and stop the execution.
- Go to `localhost:3000` and you should see a new module: `Broken sockets` 

Closes #1241
